src/pages/Encounters/EncounterList.tsx

```diff
@@ -719,8 +719,8 @@ export function EncounterList({
                         format(new Date(encounter.period.start), "PPp")}
                     </CardDescription>
                   </CardHeader>
-                  <CardContent className="">
-                    <div className="flex flex-col space-y-2">
+                  <CardContent className="md:flex-grow">
```

Comment:
why md? Let's just have 
```suggestion
                  <CardContent className="flex-grow">
```